roles/pulibrary.bibdata/molecule/default/playbook.yml

```diff
@@ -1,12 +1,21 @@
 ---
 - name: Converge
   hosts: all
+  pre_tasks:
```

Comment:
Is this needed for the Percona cluster?

Reply:
yes the container doesn't get an usable IP address otherwise. :-(